explorer/client/src/components/network/Network.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useCallback, useContext, useState } from 'react';
+
+import { NetworkContext } from '../../context';
+import { Network } from '../../utils/api/DefaultRpcClient';
+import { IS_STATIC_ENV, IS_LOCAL_ENV } from '../../utils/envUtil';
+
+import styles from './Network.module.css';
+
+export default function NetworkSelect() {
+    const [network, setNetwork] = useContext(NetworkContext);
+    const [isModuleOpen, setModuleOpen] = useState(false);
+    const [isOpenInput, setIsOpenInput] = useState(false);
+
+    const openModal = useCallback(
+        () => (isModuleOpen ? setModuleOpen(false) : setModuleOpen(true)),
+        [isModuleOpen, setModuleOpen]
+    );
+    const closeModal = useCallback(() => setModuleOpen(false), [setModuleOpen]);
+
+    const openInput = useCallback(() => {
+        setIsOpenInput(true);
+        setNetwork('http://127.0.0.1:5001');
+    }, [setIsOpenInput, setNetwork]);
+
+    const chooseNetwork = useCallback(
+        (specified: Network | string) => () => {
+            if (network !== specified) {
+                setNetwork(specified);
+                setIsOpenInput(false);
+            }
+        },
+        [network, setNetwork]
+    );
+
+    const handleTextChange = useCallback(
+        (e: React.ChangeEvent<HTMLInputElement>) =>
+            setNetwork(e.currentTarget.value),
+        [setNetwork]
+    );
+    const networkStyle = (iconNetwork: Network | 'other') =>
+        //Button text matches network or
```

Comment:
```suggestion
        // Button text matches network or
```
nit: single space after `//`